doc/source/tutorial/getting_started.rst

```diff
@@ -32,23 +31,24 @@ repository `Enzo-E / Cello github repository
 
 
 Before compiling ``Enzo-E / Cello``, you may also need to download
-and install 1.``Charm++``, 2.``HDF5``, 3.``libpng``, 3.``libboost``, and 4.``scons``:
+and install 1.``Charm++``, 2.``HDF5``, 3.``libpng``, 3.``libboost``, and 4.``cmake``:
 
 1. Install ``Charm++``
 ----------------------
 
-``Charm++`` can be `downloaded here <https://charm.cs.illinois.edu/software>`_.
+First, we install the latest `Charm++`, which now also uses `cmake` as default
+build system (as example we use the pure MPI backend).
 
-To build Charm++ on a Mac, try this:
-
-   ``./build charm++ netlrts-darwin-x86_64 -j4 --with-production``
-
-If you're running Linux, try building Charm++ using this:
-
-   ``./build charm++ netlrts-linux-x86_64   -j4  --with-production``
+..  code-block:: bash
 
-Charm++ also provides a helpful interactive script ``smart-build.pl`` that can
-be used to configure and install ``Charm++`` on numerous other platforms.
+  git clone https://github.com/UIUC-PPL/charm.git
+  cd charm
+  git checkout v7.0.0
```

Comment:
See the comments I made related to this exact snippet on the getting started pages for Frontera and Pleiades.